<style>
H1{color:Blue !important;}
H2{color:White !important; background-color: DarkBlue;}
H3{color:White !important; background-color: DarkGreen;}
</style>

## Portfolio

---

### INTRODUCTION 
- Strong technical skills including <span style="color: red;">10+</span> years of hands-on experience in <span style="color: red;">Computational Chemistry, Chemoinformatics, Molecular Dynamics, Molecular Docking, Homology Modelling, Free Energy Calculations, 3D-QSAR, Virtual screening</span>.
- Proven ability to <span style="color: red;">design, execute, analyze, interpret, and document results</span> of experiments as demonstrated by a <span style="color: blue;">16</span> publication record.
- Excellent in science communication and technical writing skills.
- Ability to achieve research targets by managing multiple projects simultaneously.
- Proven ability to work in a multidisciplinary team in highly collaborative environments.
- Extensive experience in supervising, mentoring, and training lab personnel.

### WORK EXPERIENCE
1. **`Molecular Dynamics Expert `** \
02/2019-05/2021 \
Department of Chemistry \
University of Cincinnati, OH, 45239, USA \
As a Postdoctoral Associate:
- *Expanded the use of Molecular dynamics techniques to study protein conformational changes.*
- *Perform the homology modelling to construct an atomic-resolution model of the "target" protein from its amino acid sequence and an experimental three-dimensional
structure of a related homologous protein (the "template").*
- *Participated in a collaborative research team involved in the development of the project.*
- *Provided technical assistance as subject matter expert in computational chemistry.*
- *Published <span style="color: blue;">1 first-author</span> paper and <span style="color: blue;">2 first-author</span> publications in preparation.*
- *Mentored and trained more than <span style="color: blue;">5</span> graduate and undergraduate students.*
  
2. **`Homology modelling expert`** \
11/2014-11/2018 \
SISSA \
Trieste, Italy \
As a Doctoral Student:
- *Develop web server to perform Coarse-grained molecular dynamics (MERMAID).*
- *Perform homology modeling and molecular dynamics of membrane proteins.*
- *Published <span style="color: blue;">2 first-author</span> research publications.*

3. **`Computational Chemist`** \
08/2013-03/2014 \
NIPER \
Mohali, India \
As a Junior Research Fellow:
- *Worked on Computer Aided Drug Design projects in a multidisciplinary collaborative
environment.*

4. **`CADD Expert`** \
08/2013-03/2014 \
NIPER \
Mohali, India \
As a Master's Student:
- *Developed 3D-QSAR models for the development of protein kinase inhibitors as new potential anti-tubercular agents.*
- *Perform molecular dynamics simulations to study structural insights for design of subtype-selective peroxisome proliferator activated receptor-α (PPAR-α) agonists.*
- *Collaborate in multiple cross-disciplinary projects, which results in <span style="color: blue;">13</span> research publications with <span style="color: blue;">1 first author, 2 second author and 1 book chapter</span>.*

### EDUCATION
1. **`PhD in Computational Neurobiology`** \
2014-2018 \
SISSA, Treiste, Italy
2. **`MS Pharmacoinformatics`** \
2011-2013 \
NIPER, Mohali, India
3. **`B.Pharm`** \
2007-2011 \
Government College of Pharmacy, Amravati, MH, India

### SKILLS
<p align="justify"><span style="background-color: blue; color: white">Python, R, Bash Script, LINUX</span>
<span style="background-color: red; color: white">VMD, Pymol</span>
<span style="background-color: green; color: white">Inkscape, LaTeX</span>
<span style="background-color: magenta; color: white">Chemoinformatics, Chemistry</span>
<span style="background-color: gray; color: white">GROMACS, AMBER, Autodock, Sybyl-X, Gastroplus, Modeller, Discovery Studio, Maestro, Schrodinger GLIDE, Derek Nexus, Toxtree</span>
<span style="background-color: orange; color: black">Molecular Dynamics, Coarse-grained MD, Molecular Docking, Homology Modeling, 3D-QSAR, Forcefield Parameterization, Virtual Screening</span>
<span style="background-color: teal; color: white">Interdisciplinary Coordination, Deep Science Research, Project Management, Scientific Writing, Research and development</span></p>

### ACCOMPLISHMENTS
- Oral presentation in <span style="color: red;"> American Physical Society (APS) </span>, 2020, Denver, Colorado
- Poster presentation in <span style="color: red;">Biophysical Society (BPS)</span>, 2018, SanFrancisco, California
- Best poster award winner in <span style="color: red;">The European Iron Club</span>, 2018, ETH Z¨urich
- Oral presentation in <span style="color: red;">Winter School Canazei, Applied bioinformatics</span>, 2018, Verona, Italy
- Best poster award winner in 5th <span style="color: red;">DMPK</span>, 2014, NIPER, Mohali, India
- <span style="color: red;">Developed web server</span> : **<a href="https://molsim.sci.univr.it/mangesh/index.php" target="_blank">MERMAID(Old)</a>** and **<a href="https://molsim.sci.univr.it/mermaid/main.php" target="_blank">MERMAID(New)</a>** Martini coarsE gRained MembrAne proteIn Dynamics. Tutorial: <a href="https://www.youtube.com/watch?v=hbwI7YnonEY&t=56s" target="_blank">Youtube Link</a>
- Quality assurance and Software testing: **<a href="http://www.niper.gov.in/pi_dev_tools/DruLiToWeb/DruLiTo_index.html" target="_blank">DRULITO(DrugLikeness Tool)</a>**

### GUEST REVIEWER
- Scientific Reports (SERP)
- Computational and Structural Biotechnology Journal (CSBJ) Elsevier
- Journal of Biomolecular Screening (JBS) SAGE Journals

### PUBLICATIONS
1. <span style="color: blue;">M. DAMRE</span>, A. Dayananda, R. A. Varikoti, G. Stan, and R. I. Dima, Factors underlying asymmetric dynamics of disaggregase and microtubule severing AAA+ machines," bioRχiv, nov 2020.
2. <span style="color: blue;">M. DAMRE</span>, R. A. Varikoti, and R. Dima, Molecular dynamics study of katanin oligomeres: A mt-severing enzyme," Bulletin of the American Physical Society, vol. 65, 2020.
3. <span style="color: blue;">M. DAMRE</span>, A. Marchetto, and A. Giorgetti, Mermaid: dedicated web server to prepare and run coarse-grained membrane protein dynamics," Nucleic acids research, vol. 47, no. W1, pp. W456{W461, 2019.
4. <span style="color: blue;">M. V. DAMRE</span> et al., Computational studies on membrane proteins (bovine cnga1 & mouse tspo)," Ph.D. dissertation, SISSA, Trieste, Italy, 2018.
5. J. Zeng, R. Guareschi, <span style="color: blue;">M. DAMRE</span>, R. Cao, A. Kless, B. Neumaier, A. Bauer, A. Giorgetti, P. Carloni, and G. Rossetti, Structural prediction of the dimeric form of the mammalian translocator membrane protein tspo: a key target for brain diagnostics," International journal of molecular sciences, vol. 19, no. 9, p. 2588, 2018.
6. <span style="color: blue;">M. V. DAMRE</span>, A. Giorgetti, and V. Torre, Gating mechanism investigation in homotetramer cnga1 ion channel by coarse-grained molecular dynamics simulation," Biophysical Journal, vol. 114, no. 3, p. 128a, 2018.
7. <span style="color: blue;">M. V. DAMRE</span>, Design and pharmacokinetic profiling of selective ppar-α agonists: A molecular docking and md simulation approach," Ph.D. dissertation, National Institute of Pharmaceutical Education and Research, 2013.
8. R. P. Gangwal, G. V. Dhoke, <span style="color: blue;">M. V. DAMRE</span>, V. Sharma, and A. T. Sangamwar, Design of novel cytochrome bc1 inhibitors: A molecular modelling approach," DMPK2013, 2013.
9. R. P. Gangwal, <span style="color: blue;">M. V. DAMRE</span>, and A. T. Sangamwar, Overview and recent advances in qsar sudies," in Chemometrics Applications and Research: QSAR in Medicinal Chemistry. CRC Press, 2016, p. 1.
10. N. Soumya, H. Tandan, <span style="color: blue;">M. V. DAMRE</span>, R. P. Gangwal, A. T. Sangamwar, and S. Singh, Leucine-684: A conserved residue of an amp-acetyl coa synthetase (acecs) from leishmania donovani is involved in substrate recognition, catalysis and acetylation," Gene, vol. 580, no. 2, pp. 125{133, 2016.
11. R. P. Gangwal, <span style="color: blue;">M. V. DAMRE</span>, N. R. Das, G. V. Dhoke, A. Bhadauriya, R. A. Varikoti, S. S. Sharma, and A. T. Sangamwar, Structure based virtual screening to identify selective phosphodiesterase 4b inhibitors," Journal of Molecular Graphics and Modelling, vol. 57, pp. 89{98, 2015.
12. R. P. Gangwal, <span style="color: blue;">M. V. DAMRE</span>, N. R. Das, S. S. Sharma, and A. T. Sangamwar, Biological evaluation and structural insights for design of subtype-selective peroxisome proliferator activated receptor-α (ppar-α) agonists," Bioorganic & medicinal chemistry letters, vol. 25, no. 2, pp. 270{275, 2015.
13. K. Khandelwal, R. P. Gangwal, U. Singh, R. Prajapati, <span style="color: blue;">M. V. DAMRE</span>, and A. T. Sangamwar, Computational insights into the active site of human breast cancer resistance protein (bcrp/abcg2): a similarity search approach," Medicinal Chemistry Research, vol. 23, no. 11, pp. 4657{4668, 2014
14. D. Uppalapati, N. R. Das, R. P. Gangwal, <span style="color: blue;">M. V. DAMRE</span>, A. T. Sangamwar, and S. S. Sharma, Neuroprotective potential of peroxisome proliferator activated receptor-α agonist in cognitive impairment in parkinson’s disease: Behavioral, biochemical, and pbpk profile," PPAR research, vol. 2014, 2014
15. N. R Das, R. P Gangwal, <span style="color: blue;">M. V DAMRE</span>, A. T Sangamwar, and S. S Sharma, A ppar-β/δ agonist is neuroprotective and decreases cognitive impairment in a rodent model of parkinson’s disease," Current neurovascular research, vol. 11, no. 2, pp. 114{124, 2014
16. R. P. Gangwal, N. R. Das, K. Thanki, <span style="color: blue;">M. V. DAMRE</span>, G. V. Dhoke, S. S. Sharma, S. Jain, and A. T. Sangamwar, Identification of p38α map kinase inhibitors by pharmacophore based virtual screening," Journal of Molecular Graphics and Modelling, vol. 49, pp. 18{24, 2014
17. R. P. Gangwal, G. V. Dhoke, <span style="color: blue;">M. V. DAMRE</span>, K. Khandelwal, and A. T. Sangamwar, Structure-based virtual screening and molecular dynamic simulation studies to identify novel cytochrome bc1 inhibitors as antimalarial agents," Journal of Computational Medicine, vol. 2013, 2013
18. <span style="color: blue;">M. V. DAMRE</span>, R. P. Gangwal, G. V. Dhoke, M. Lalit, D. Sharma, K. Khandelwal, and A. T. Sangamwar, 3d-qsar and molecular docking studies of amino-pyrimidine derivatives as pknb inhibitors," Journal of the Taiwan Institute of Chemical Engineers, vol. 45, no. 2, pp. 354{364, 2014
19. M. Lalit, R. P. Gangwal, G. V. Dhoke, <span style="color: blue;">M. V. DAMRE</span>, K. Khandelwal, and A. T. Sangamwar, A combined pharmacophore modeling, 3d-qsar and molecular docking study of substituted bicyclo-[3.3. 0] oct-2-enes as liver receptor homolog-1 (lrh-1) agonists," Journal of Molecular Structure, vol. 1049, pp. 315{325, 2013
20. R. P. Gangwal, A. Bhadauriya, <span style="color: blue;">M. V. DAMRE</span>, G. V. Dhoke, and A. T. Sangamwar, p38 mitogen-activated protein kinase inhibitors: a review on pharmacophore mapping and qsar studies," Current topics in medicinal chemistry, vol. 13, no. 9, pp. 1015{1035, 2013
21. A. Bhadauriya, G. V. Dhoke, R. P. Gangwal, <span style="color: blue;">M. V. DAMRE</span>, and A. T. Sangamwar, Identification of dual acetylcoa carboxylases 1 and 2 inhibitors by pharmacophore based virtual screening and molecular docking approach," Molecular diversity, vol. 17, no. 1, pp. 139{149, 2013
22. U. Singh, R. P. Gangwal, G. V. Dhoke, R. Prajapati, <span style="color: blue;">M. DAMRE</span>, and A. T. Sangamwar, 3d-qsar and molecular docking analysis of (4-piperidinyl)-piperazines as acetyl-coa carboxylases inhibitors," Arabian Journal of Chemistry, 2012<br>

---
![Jetpacktocat](https://octodex.github.com/images/jetpacktocat.png)

<!-- Remove above link if you don't want to attibute -->
