<style>
H1{color:Blue !important;}
H2{color:White !important; background-color: DarkBlue;}
H3{color:White !important; background-color: DarkGreen;}
</style>

## Portfolio

---

### INTRODUCTION 
<p align="justify">Highly motivated Postdoctoral Researcher conducts Computational Chemistry research and participates in brainstorming topics that demonstrate experience with complex data analysis and high-performance computing. Dedicated, versatile computational chemist, author and co-author of international publications in peer-reviewed journals with extensive research experience in molecular modeling. Recent PhD in Neurobiology (Computational) with effective verbal and written communication skills to deliver excellent presentations and prepare written reports. Broad range of knowledge in advanced computational techniques. Develop and apply computational methods in the fields of biophysics, chemistry and other computationally intensive areas. Optimize and parallelize scientific programs for conventional clusters, supercomputers and GPU accelerated computers. Solve challenging problems in computational biophysics/ chemistry field through varied skills and interdisciplinary science knowledge.</p>

### WORK EXPERIENCE
1. **PostDoctoral Researcher**<br>
02/2019-continue...<br>
Department of Chemistry<br>
University of Cincinnati, OH, 45239, USA<br>
2. **Junior Research Fellow**<br>
08/2013-03/2014<br>
Department of Pharmacoinformatics<br>
NIPER, Mohali, India<br>

### EDUCATION
1. **PhD in Computational Neurobiology**<br>
   2014-2018<br>
   SISSA, Treiste, Italy.<br>
2. **MS Pharmacoinformatics**<br>
   2011-2013<br>
   NIPER, Mohali, India.<br>
3. **B.Pharm**<br>
   2007-2011<br>
   Government College of Pharmacy, Amravati, MH, India.<br>

### SKILLS
<p align="justify"><span style="background-color: blue; color: white">Python, R, Bash Script, LINUX</span>
<span style="background-color: red; color: white">VMD, Pymol</span>
<span style="background-color: green; color: white">Inkscape, LaTeX</span>
<span style="background-color: magenta; color: white">Chemoinformatics, Chemistry</span>
<span style="background-color: gray; color: white">GROMACS, AMBER, Autodock, Sybyl-X, Gastroplus, Modeller, Discovery Studio, Maestro, Schrodinger GLIDE, Derek Nexus, Toxtree</span>
<span style="background-color: orange; color: black">Molecular Dynamics, Coarse-grained MD, Molecular Docking, Homology Modeling, 3D-QSAR, Forcefield Parameterization, Virtual Screening</span>
<span style="background-color: teal; color: white">Interdisciplinary Coordination, Deep Science Research, Project Management, Scientific Writing, Research and development</span></p>

### ACCOMPLISHMENTS
- Oral presentation in <span style="color: red;"> American Physical Society (APS) </span>, 2020, Denver, Colorado<br>
- Poster presentation in <span style="color: red;">Biophysical Society (BPS)</span>, 2018, SanFrancisco, California<br>
- Best poster award winner in <span style="color: red;">The European Iron Club</span>, 2018, ETH Z¨urich<br>
- Oral presentation in <span style="color: red;">Winter School Canazei, Applied bioinformatics</span>, 2018, Verona, Italy<br>
- Best poster award winner in 5th <span style="color: red;">DMPK</span>, 2014, NIPER, Mohali, India<br>
- <span style="color: red;">Developed web server</span> : (**<a href="https://molsim.sci.univr.it/mangesh/index.php" target="_blank">MERMAID</a>**) Martini Coarse Grained Membrane Protein Dynamics<br>

### PUBLICATIONS
<p align="justify">
   1. <span style="color: blue;">M. DAMRE</span>, A. Dayananda, R. A. Varikoti, G. Stan, and R. I. Dima, Factors underlying asymmetric dynamics of disaggregase and microtubule severing AAA+ machines," bioRχiv, nov 2020.
   2. <span style="color: blue;">M. DAMRE</span>, R. A. Varikoti, and R. Dima, Molecular dynamics study of katanin oligomeres: A mt-severing enzyme," Bulletin of the American Physical Society, vol. 65, 2020.
   3. <span style="color: blue;">M. DAMRE</span>, A. Marchetto, and A. Giorgetti, Mermaid: dedicated web server to prepare and run coarse-grained membrane protein dynamics," Nucleic acids research, vol. 47, no. W1, pp. W456{W461, 2019.
   4. <span style="color: blue;">M. V. DAMRE</span> et al., Computational studies on membrane proteins (bovine cnga1 & mouse tspo)," Ph.D. dissertation, SISSA, Trieste, Italy, 2018.
   5. J. Zeng, R. Guareschi, <span style="color: blue;">M. DAMRE</span>, R. Cao, A. Kless, B. Neumaier, A. Bauer, A. Giorgetti, P. Carloni, and G. Rossetti, Structural prediction of the dimeric form of the mammalian translocator membrane protein tspo: a key target for brain diagnostics," International journal of molecular sciences, vol. 19, no. 9, p. 2588, 2018.
   6. <span style="color: blue;">M. V. DAMRE</span>, A. Giorgetti, and V. Torre, Gating mechanism investigation in homotetramer cnga1 ion channel by coarse-grained molecular dynamics simulation," Biophysical Journal, vol. 114, no. 3, p. 128a, 2018.
   7. <span style="color: blue;">M. V. DAMRE</span>, Design and pharmacokinetic profiling of selective ppar-α agonists: A molecular docking and md simulation approach," Ph.D. dissertation, National Institute of Pharmaceutical Education and Research, 2013.
   8. R. P. Gangwal, G. V. Dhoke, <span style="color: blue;">M. V. DAMRE</span>, V. Sharma, and A. T. Sangamwar, Design of novel cytochrome bc1 inhibitors: A molecular modelling approach," DMPK2013, 2013.
   9. R. P. Gangwal, <span style="color: blue;">M. V. DAMRE</span>, and A. T. Sangamwar, Overview and recent advances in qsar sudies," in Chemometrics Applications and Research: QSAR in Medicinal Chemistry. CRC Press, 2016, p. 1.
   10. N. Soumya, H. Tandan, <span style="color: blue;">M. V. DAMRE</span>, R. P. Gangwal, A. T. Sangamwar, and S. Singh, Leucine-684: A conserved residue of an amp-acetyl coa synthetase (acecs) from leishmania donovani is involved in substrate recognition, catalysis and acetylation," Gene, vol. 580, no. 2, pp. 125{133, 2016.
   11. R. P. Gangwal, <span style="color: blue;">M. V. DAMRE</span>, N. R. Das, G. V. Dhoke, A. Bhadauriya, R. A. Varikoti, S. S. Sharma, and A. T. Sangamwar, Structure based virtual screening to identify selective phosphodiesterase 4b inhibitors," Journal of Molecular Graphics and Modelling, vol. 57, pp. 89{98, 2015.
   12. R. P. Gangwal, <span style="color: blue;">M. V. DAMRE</span>, N. R. Das, S. S. Sharma, and A. T. Sangamwar, Biological evaluation and structural insights for design of subtype-selective peroxisome proliferator activated receptor-α (ppar-α) agonists," Bioorganic & medicinal chemistry letters, vol. 25, no. 2, pp. 270{275, 2015.
   13. K. Khandelwal, R. P. Gangwal, U. Singh, R. Prajapati, <span style="color: blue;">M. V. DAMRE</span>, and A. T. Sangamwar, Computational insights into the active site of human breast cancer resistance protein (bcrp/abcg2): a similarity search approach," Medicinal Chemistry Research, vol. 23, no. 11, pp. 4657{4668, 2014
   14. D. Uppalapati, N. R. Das, R. P. Gangwal, <span style="color: blue;">M. V. DAMRE</span>, A. T. Sangamwar, and S. S. Sharma, Neuroprotective potential of peroxisome proliferator activated receptor-α agonist in cognitive impairment in parkinson’s disease: Behavioral, biochemical, and pbpk profile," PPAR research, vol. 2014, 2014
   15. N. R Das, R. P Gangwal, <span style="color: blue;">M. V DAMRE</span>, A. T Sangamwar, and S. S Sharma, A ppar-β/δ agonist is neuroprotective and decreases cognitive impairment in a rodent model of parkinson’s disease," Current neurovascular research, vol. 11, no. 2, pp. 114{124, 2014
   16. R. P. Gangwal, N. R. Das, K. Thanki, <span style="color: blue;">M. V. DAMRE</span>, G. V. Dhoke, S. S. Sharma, S. Jain, and A. T. Sangamwar, Identification of p38α map kinase inhibitors by pharmacophore based virtual screening," Journal of Molecular Graphics and Modelling, vol. 49, pp. 18{24, 2014
   17. R. P. Gangwal, G. V. Dhoke, <span style="color: blue;">M. V. DAMRE</span>, K. Khandelwal, and A. T. Sangamwar, Structure-based virtual screening and molecular dynamic simulation studies to identify novel cytochrome bc1 inhibitors as antimalarial agents," Journal of Computational Medicine, vol. 2013, 2013
   18. <span style="color: blue;">M. V. DAMRE</span>, R. P. Gangwal, G. V. Dhoke, M. Lalit, D. Sharma, K. Khandelwal, and A. T. Sangamwar, 3d-qsar and molecular docking studies of amino-pyrimidine derivatives as pknb inhibitors," Journal of the Taiwan Institute of Chemical Engineers, vol. 45, no. 2, pp. 354{364, 2014
   19. M. Lalit, R. P. Gangwal, G. V. Dhoke, <span style="color: blue;">M. V. DAMRE</span>, K. Khandelwal, and A. T. Sangamwar, A combined pharmacophore modeling, 3d-qsar and molecular docking study of substituted bicyclo-[3.3. 0] oct-2-enes as liver receptor homolog-1 (lrh-1) agonists," Journal of Molecular Structure, vol. 1049, pp. 315{325, 2013
   20. R. P. Gangwal, A. Bhadauriya, <span style="color: blue;">M. V. DAMRE</span>, G. V. Dhoke, and A. T. Sangamwar, p38 mitogen-activated protein kinase inhibitors: a review on pharmacophore mapping and qsar studies," Current topics in medicinal chemistry, vol. 13, no. 9, pp. 1015{1035, 2013
   21. A. Bhadauriya, G. V. Dhoke, R. P. Gangwal, <span style="color: blue;">M. V. DAMRE</span>, and A. T. Sangamwar, Identification of dual acetylcoa carboxylases 1 and 2 inhibitors by pharmacophore based virtual screening and molecular docking approach," Molecular diversity, vol. 17, no. 1, pp. 139{149, 2013
   22. U. Singh, R. P. Gangwal, G. V. Dhoke, R. Prajapati, <span style="color: blue;">M. DAMRE</span>, and A. T. Sangamwar, 3d-qsar and molecular docking analysis of (4-piperidinyl)-piperazines as acetyl-coa carboxylases inhibitors," Arabian Journal of Chemistry, 2012</p><br>

---
![Jetpacktocat](https://octodex.github.com/images/jetpacktocat.png)

<!-- Remove above link if you don't want to attibute -->
